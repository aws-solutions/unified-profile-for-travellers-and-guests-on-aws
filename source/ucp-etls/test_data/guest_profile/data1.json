{    "modelVersion": "1.0",    "id": "1185539038",    "lastUpdatedOn": "2021-07-10T01:59:28.740702058Z",    "createdOn": "2021-05-10T01:59:28.740702058Z",    "lastUpdatedBy": "Liza Gislason",    "createdBy": "Gavin Quitzon",    "emails": [        {            "type": "personal",            "address": "ronaldoebert@howe.info",            "primary": false        }    ],    "phones": [],    "addresses": [        {            "type": "mailing",            "line1": "3846 North Locks port",            "line2": "more content line 2",            "line3": "more content line 3",            "line4": "more content line 4",            "city": "Birmingham",            "stateProvince": {                "code": "WA",                "name": ""            },            "postalCode": "52336",            "country": {                "code": "SZ",                "name": ""            },            "primary": false        },        {            "type": "mailing",            "line1": "4318 Forge fort",            "line2": "more content line 2",            "line3": "more content line 3",            "line4": "more content line 4",            "city": "Cincinnati",            "stateProvince": {                "code": "VT",                "name": ""            },            "postalCode": "30639",            "country": {                "code": "AO",                "name": ""            },            "primary": true        }    ],    "honorific": "Dr",    "firstName": "Era",    "middleName": "Louie",    "lastName": "Hand",    "gender": "other",    "pronoun": "they",    "dateOfBirth": "2019-08-30",    "language": {        "code": "tg",        "name": ""    },    "nationality": {        "code": "CX",        "name": ""    },    "jobTitle": "Supervisor",    "parentCompany": "optiGov",    "loyaltyPrograms": [        {            "id": "9679330599",            "programName": "reward",            "points": 270841.0,            "pointUnit": "points",            "pointsToNextLevel": 9629.0,            "level": "gold",            "joined": "2021-05-10T01:59:28.740702058Z"        },        {            "id": "5276878896",            "programName": "elite",            "points": 466491.0,            "pointUnit": "reward points",            "pointsToNextLevel": 6528.0,            "level": "silver",            "joined": "2021-05-10T01:59:28.740702058Z"        }    ]}