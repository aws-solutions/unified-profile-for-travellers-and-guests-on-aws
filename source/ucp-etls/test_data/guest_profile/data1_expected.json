{"guest_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "last_updated": "2022-03-20T11:58:02.529353371Z", "created_on": "2021-12-20T11:58:02.529353371Z", "last_updated_by": "Vincent Mueller", "created_by": "Tyrel Torp", "honorific": "Cllr", "first_name": "Abel", "middle_name": "Hoyt", "last_name": "Wilkinson", "gender": "other", "pronoun": "they", "date_of_birth": "1922-05-17", "job_title": "Coordinator", "company": "Food+Tech Connect", "nationality_code": "CR", "nationality_name": "", "language_code": "ae", "language_name": "", "email_business": "", "email": "", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "shipping", "traveller_id": "6494295187", "last_updated_partition": "2022/03/20/11/58/02/529353371Z"}], "common_email_recs": [], "common_phone_recs": [], "hotel_loyalty_recs": [{"object_type": "air_loyalty", "model_version": "1.0", "last_updated": "2022-03-20T11:58:02.529353371Z", "last_updated_by": "Vincent Mueller", "id": "7690224677", "program_name": "elite", "points": 209188.0, "units": "miles", "points_to_next_level": 585.0, "level": "platinium", "joined": "2021-12-20T11:58:02.529353371Z", "traveller_id": "6494295187", "last_updated_partition": "2022/03/20/11/58/02/529353371Z"}]}